docs/snippets/ov_extensions.cpp

```diff
@@ -119,20 +142,52 @@ core.add_extension(ov::frontend::OpExtension<>("Relu", "MyRelu"));
 core.add_extension(ov::frontend::OpExtension<CustomOperation>());
 //! [frontend_extension_CustomOperation_as_is]
 
+//! [frontend_extension_CustomOperation_as_is_paddle]
+core.add_extension(ov::frontend::OpExtension<CustomOperation>({"A", "B", "C"}, {"X", "Y"}));
+//! [frontend_extension_CustomOperation_as_is_paddle]
+
 //! [frontend_extension_CustomOperation_rename]
 core.add_extension(ov::frontend::OpExtension<CustomOperation>(
     std::map<std::string, std::string>{ {"attr1", "fw_attr1"}, {"attr2", "fw_attr2"} },
     {}
 ));
 //! [frontend_extension_CustomOperation_rename]
 
+//! [frontend_extension_CustomOperation_rename_paddle]
+core.add_extension(ov::frontend::OpExtension<CustomOperation>(
+    {"A", "B", "C"},
+    {"X", "Y"},
+    std::map<std::string, std::string>{ {"attr1", "fw_attr1"}, {"attr2", "fw_attr2"} },
+    {}
+));
+//! [frontend_extension_CustomOperation_rename_paddle]
+
+
 //! [frontend_extension_CustomOperation_rename_set]
 core.add_extension(ov::frontend::OpExtension<CustomOperation>(
     std::map<std::string, std::string>{ {"attr1", "fw_attr1"} },
     { {"attr2", 5} }
 ));
 //! [frontend_extension_CustomOperation_rename_set]
 
+//! [frontend_extension_CustomOperation_rename_set_paddle]
+core.add_extension(ov::frontend::OpExtension<CustomOperation>(
+    {"A", "B", "C"},
+    {"X", "Y"},
+    std::map<std::string, std::string>{ {"attr1", "fw_attr1"} },
+    { {"attr2", 5} }
+));
+//! [frontend_extension_CustomOperation_rename_set_paddle]
+
+{
+//! [frontend_extension_framework_map_CustomElu_mapping]
+auto extension = std::make_shared<ov::frontend::OpExtension<CustomElu>>("aten::elu",
+                                                                        std::map<std::string, size_t>{{"alpha", 1}},
+                                                                        std::map<std::string, ov::Any>{{"beta", 1.0f}});
```

Comment:
How can we avoid explicitly writing `std::map<std::string, ...>`? Looks ugly, initially we rely on automatically deduced types and it was nice.

Reply:
I couldn't find a way to avoid that. The code simply doesn't compile.

Reply:
Looks to be too many overloads. @itikhono, @andrei-kochin, who can cover it from the core side? It doesn't look good and should be improved.

Reply:
There are lots of such ugly examples in the document, where we need to write the type randomly. It is bad from UX side. Please create a ticket to cover it in the next release.

Reply:
CVS-110253